include/rppt_tensor_effects_augmentations.h

```diff
@@ -165,6 +165,25 @@ RppStatus rppt_non_linear_blend_gpu(RppPtr_t srcPtr1, RppPtr_t srcPtr2, RpptDesc
 
 /******************** glitch ********************/
 
+// Performs a glitch augmentation for a batch of images
+
+// *param[in] srcPtr source tensor memory
+// *param[in] srcDescPtr source tensor descriptor
+// *param[out] dstPtr destination tensor memory
+// *param[in] dstDescPtr destination tensor descriptor
+// *param[in] xOffsetR Array containing an Rpp32u x offset value for the r-channel pixels in each image in the batch (x_offset_r[n] >= 0)
```

Comment:
- Please change the name from x_offset_r to xOffsetR. Applicable to all such instances in this file
- No need to mention the datatype of the parameter. Please refer other comments in this file 
`// *param[in] stdDevTensor standard deviation values to quantify non-linearity in the blend (1D tensor of size batchSize with stdDevTensor[n] > 0 for each image in batch)`

This can be updated as xOffset value for R channel (1D tensor of size batchsize with xOffserR[n] > 0)

Reply:
Done